esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -141,10 +144,12 @@ define([
             var protectedCheckbox = registry.byId(this.id + "Protected");
             var context = this;
             this.wu.update({
-                Scope: dom.byId(context.id + "Scope").value,
-                Description: dom.byId(context.id + "Description").value,
+                State: dom.byId(this.id + "State").innerHTML,
```

Comment:
Yes, the API requires both states. /WsWorkunits/WUUpdate?form

State:	
StateOrig: